From 85c091652669b935dfc46defc8a2de0154f4ced7 Mon Sep 17 00:00:00 2001
From: James Green <green@ms1p.org>
Date: Tue, 22 Sep 2026 13:25:22 +0200
Subject: [PATCH 4/6] Do not use mstore and toml-f nor build app

- mstore and toml-f cause issues for some compilers
e.g. nvhpc and classic intel compilers
- app uses toml-f so it shouldn't be built
---
 CMakeLists.txt | 14 +++-----------
 1 file changed, 3 insertions(+), 11 deletions(-)

diff --git a/CMakeLists.txt b/CMakeLists.txt
index 0819a5b..209f6aa 100644
--- a/CMakeLists.txt
+++ b/CMakeLists.txt
@@ -37,20 +37,10 @@ endif()
 if(NOT TARGET "mctc-lib::mctc-lib")
   find_package("mctc-lib" REQUIRED)
 endif()
-if(NOT TARGET "mstore::mstore" AND WITH_TESTS)
-  find_package("mstore" REQUIRED)
-endif()
-if(NOT TARGET "toml-f::toml-f")
-  find_package("toml-f" REQUIRED)
-endif()
 set(
   lib-deps
   "mctc-lib::mctc-lib"
 )
-set(
-  exe-deps
-  "toml-f::toml-f"
-)
 
 # Collect source of the project
 set(srcs)
@@ -92,7 +82,9 @@ target_include_directories(
 )
 
 # Add example application
-add_subdirectory("app")
+# JAG: 11/10/24: Don't build app as it needs toml-f
+# which I have removed as it causes issues with intel 2020 compilers
+#add_subdirectory("app")
 
 # Export targets for other projects
 add_library("${PROJECT_NAME}" INTERFACE)
-- 
2.43.0

